src/components/IndexContents.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import TableOfContentsContainer from './TableOfContentsContainer';
+import { Link } from '@newrelic/gatsby-theme-newrelic';
+
+const IndexContents = ({ nav }) => {
+  const { pages } = nav;
+
+  return (
+    <TableOfContentsContainer>
+      {pages.map((page) => (
+        <TableOfContents key={page.title} root={page} />
+      ))}
+    </TableOfContentsContainer>
+  );
+};
+
+IndexContents.propTypes = {
+  nav: PropTypes.object,
```

Comment:
[nit] we might want to add a `shape` and `isRequired` to this.

Reply:
You're probably right 😄 . I was being lazy 😅

Reply:
Addressed in 106aa6d